server/services/omnichannel-voip/service.ts

```diff
@@ -1,10 +1,84 @@
-import { ILivechatVoipService } from '../../sdk/types/IOmnichannelVoipService';
+import { Db } from 'mongodb';
+import _ from 'underscore';
+
+import { IOmnichannelVoipService } from '../../sdk/types/IOmnichannelVoipService';
 import { ServiceClass } from '../../sdk/types/ServiceClass';
+import { IVoipConnectorResult } from '../../../definition/IVoipConnectorResult';
+import { IVoipExtensionBase } from '../../../definition/IVoipExtension';
+import { Logger } from '../../lib/logger/Logger';
+import { Voip } from '../../sdk';
+import { IAgentExtensionMap, IOmnichannelVoipServiceResult } from '../../../definition/ILivechatVoipServiceResult';
+/**
+ * Note (Amol) :
+ * Why cant we get the direct access to the model here and have to work with
+ * raw model? This is still a server code. So ideally it should have access to the model
+ * like the REST apis have.
+ *
+ * Check this during the code review and after code review is done, delete this comment.
```

Comment:
Is it okay to remove this comment now? Let me know if you have any more doubts regarding models

Reply:
Removed the comment.